pkg/src/get-app-config.go

```diff
@@ -105,5 +106,12 @@ func main() {
 		os.Exit(1)
 	}
 	toString, err := json.Marshal(config)
```

Comment:
Can you please add an error check for this returned error? I know this isn't code that you changed, but it would be nice to add it here.